### PR TITLE
Some grammatical and stylistic changes in the README

### PR DESCRIPTION
**Pull request by bouk**
@sirupsen This has a bunch of stylistic changes, you don't have to agree with all of them :). It's kind of hard to read in places and I tried to clear it up.

### Some more notes

From the standpoint of a non-C developer, the terms `ECONNREFUSED` and `ECONNRESET` might be unfamiliar

"convergence time" is kind of vague

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,25 +144,23 @@ slow resources. But it is by no means a magic wand that solves all your latency
...
-browsing a store and you are signed in, Shopify stores your session in a session
-store. If the session store becomes unavailable, the data driver will start
-throwing driver errors. We rescue these exceptions and simply disable all
-customer sign in functionality on the store until the session store is back
-online.
+This is better illustrated with a real world example from Shopify. When you are
+browsing a store while signed in, Shopify stores your session in Redis.
+If Redis becomes unavailable, the driver will start throwing exceptions.
+We rescue these exceptions and simply disable all customer sign in functionality
+on the store until Redis is back online.
```
Comment (bouk) [inline README.md]: I changed this to just use "Redis" instead of "store/session store" because it was a bit verbose

File: README.md [hunk 2]
```diff
@@ -144,25 +144,23 @@ slow resources. But it is by no means a magic wand that solves all your latency
...
 
-This is better illustrated with a real world example from Shopify. If you are
-browsing a store and you are signed in, Shopify stores your session in a session
-store. If the session store becomes unavailable, the data driver will start
-throwing driver errors. We rescue these exceptions and simply disable all
-customer sign in functionality on the store until the session store is back
-online.
+This is better illustrated with a real world example from Shopify. When you are
+browsing a store while signed in, Shopify stores your session in Redis.
+If Redis becomes unavailable, the driver will start throwing exceptions.
```
Comment (fw42) [inline README.md]: exceptions are raised, not thrown

File: README.md (hunk 1 above)
Comment (Sirupsen) [inline README.md]: Thank you!

**Comment by Sirupsen** [comment]
Agree with all your changes, thanks a lot for doing that @bouk!

**Comment by Sirupsen** [comment]
I merged a little quickly, just commit fixes to master from e.g. @fw42 or whoever else comments.

File: README.md (hunk 2 above)
Comment (Sirupsen) [inline README.md]: fixed in master

**Pull request merged**